### DISCUSSION TITLE
How to find C/2022 ZTF on Stellarium

### DISCUSSION THREAD

**Discussion by Frankie5angels1217** [Stellarium/stellarium#3030] [General]
Hi, I don't think that comet c/2022 is in the current Comet database ( ver. 1.2). Is there a way to add c/2022 data into Stellarium?

**Comment by gzotti** [comment]
Yes. User Guide 13.7. Or https://github.com/Stellarium/stellarium/wiki/FAQ#adding-a-comet-to-stellarium-with-the-solar-system-editor-plug-in.
